CMakeLists.txt

```diff
@@ -33,9 +33,6 @@ else()
 endif()
 csfml_set_option(CSFML_LINK_SFML_STATICALLY ${LINK_STATICALLY_DEFAULT} BOOL "TRUE to link to a static version of SFML, FALSE to link dynamically")
 
-# disable the rpath stuff
-set(CMAKE_SKIP_BUILD_RPATH TRUE)
```

Comment:
https://github.com/SFML/SFML/commit/a991fe8e4db5104133ff3cefd7ec8c38339dfeae#diff-1e7de1ae2d059d21e1dd75d5812d5a34b0222cef273b7c3a2af62eb747f9d20aR44

I tried to track down when this was added and got as far back as to SFML's original CMake support added 14 years ago. I can't find any rationale for this line which makes more comfortable removing it.